payload_bay_micro.ino

```diff
@@ -11,21 +11,48 @@ unsigned long lastRadioTime = 0;
 
 Altimeter alt;
 
-void sendToRadio(float altitude) {
-  char buf[11];
+constexpr float mapFloat(
+  float value,
+  float xMin, float xMax,
+  float yMin, float yMax
+) noexcept {
+  return (yMax - yMin) / (xMax - xMin) * (value - xMin) + yMin;
```

Comment:
Could you please explain the equation here? I don't understand it, and I'd like to

Reply:
So if you have two points (x<sub>1</sub>, y<sub>1</sub>) and (x<sub>2</sub>, y<sub>2</sub>), the equation is

$$y - y_1 = \frac{y_2 - y_1}{x_2 - x_1} \left(x - x_1\right)$$

Here, I've called them (xMin, yMin) and (xMax, yMax) instead, but the equation is the same.

Reply:
Got it! So you're rearranging to solve for `y` to get it into a form of `y = mx + b` - okay that makes sense

Reply:
The fraction is the slope (rise over run), and the $-y_1$ and $-x_1$ align it with one of the points -- it could just as easily be $-y_2$ and $-x_2$.